packages/web/src/common/store/lineup/sagas.js

```diff
@@ -25,15 +25,37 @@ import {
   takeLatest
 } from 'redux-saga/effects'
 
-import { getToQueue } from 'store/queue/sagas'
-import { isMobile } from 'utils/clientUtil'
+import { getToQueue } from 'common/store/queue/sagas'
 
 const { getSource, getUid, getPositions } = queueSelectors
 const { getUid: getCurrentPlayerTrackUid } = playerSelectors
 const { getUsers } = cacheUsersSelectors
 const { getTrack, getTracks } = cacheTracksSelectors
 const { getCollection } = cacheCollectionsSelectors
 
+const IS_NATIVE_MOBILE = process.env.REACT_APP_NATIVE_MOBILE
```

Comment:
note: duplicating here since it was defined in a non-common file imported by 90 other files

Reply:
maybe put this in a comment? (the part about it being ignored in native)

Reply:
sorry just thought about this more, you could also have the caller pass `isMobile` to the `fetchLineupMetadatasAsync` function? since thats the only place it's used

Reply:
yeah so caller is in this same file, `watchFetchLineupMetadata` which is just watching for the action to be dispatched, so i think the main way we can get env specific stuff here is to have the "isMobile" thing in the context. I went ahead and short-circuted this isMobile thing by adding the `isNativeMobile` context check which will ensure it never gets called with new system, but is compatible with old. what do you think?